Khan/wonder-blocks
# Change darkBlue value in Wonder Blocks Color

## Summary:
Update the value of `darkBlue` from #0a2a66 to #0b2149.
Also, update the value of the `darkblue` background color for
Storybook accordingly.

Note that the manually written darkBlue colors in webapp will also
need to be updated in a different PR.

Issue: https://khanacademy.atlassian.net/browse/WB-1381

## Before:
<img width="697" alt="Screen Shot 2022-08-03 at 5 07 20 PM" src="https://user-images.githubusercontent.com/13231763/182735763-3c548966-05b2-4427-840b-fcbce6f6bb3b.png">

## After:
<img width="710" alt="Screen Shot 2022-08-03 at 5 07 25 PM" src="https://user-images.githubusercontent.com/13231763/182735772-4f118ee7-33c4-4fed-b645-37fd2bd3bbb2.png">


## Test plan:
- Type `0a2a66` into the VS Code search back and confirm
  that there are no files (except for md files for now)
  that contain this value.
- Go to the Dark story under Button and check that the background
  is using the new blue.
